### PR TITLE
[Documentation] minc_deletion.pl

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Generated using:
pod2markdown uploadNeuroDB/minc_deletion.pl docs/scripts_md/minc_deletion.md

User can type:
perldoc uploadNeuroDB/minc_deletion.pl
to get the documentation at the terminal

### CODE REVIEW

File: docs/scripts_md/minc_deletion.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# NAME
+
+minc\_deletion.pl -- this script deletes files records from the database, and
+archives the actual files. Files to be deleted can be specified either based on
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: replace "actual files" by "backend files stored in /data/project/data"

File: docs/scripts_md/minc_deletion.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+\-series\_uid : the series UID of the file to be deleted
+
+\-fileid     : the file ID of the file to be deleted
+
+# DESCRIPTION
+
+The program does the following:
+
+Deletes minc files from Loris by:
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: minc and loris should be all upper case letters

File: docs/scripts_md/minc_deletion.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+\-profile    : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-series\_uid : the series UID of the file to be deleted
+
+\-fileid     : the file ID of the file to be deleted
+
+# DESCRIPTION
+
+The program does the following:
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: Remove this sentence (I think?)

File: docs/scripts_md/minc_deletion.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Deletes minc files from Loris by:
+  - Moving the existing files (.mnc .nii .jpg .header .raw\_byte.gz) to an
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: add a comma between file types.

File: docs/scripts_md/minc_deletion.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    archive directory
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: might be worth mentioning where is this archive directory here

File: docs/scripts_md/minc_deletion.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  - Deleting all related data from 2 database tables:
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: Rephrase to "Deleting data from parameter\_file & files"

File: docs/scripts_md/minc_deletion.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    parameter\_file & files
+  - Deleting data from files\_qcstatus & feedback\_mri\_comments
+    database tables if the -delqcdata is set. In most cases
```
Comment (cmadjar) [inline docs/scripts_md/minc_deletion.md]: add the word "option" after "-delqcdata".
Also, maybe have "-delqcdata" display as code?

File: docs/scripts_md/minc_deletion.md (hunk 1 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/minc_deletion.md]: ok! I added assembly in the end since we are deleting MINC files.

**Comment by MounaSafiHarab** [comment]
@cmadjar 

done I think!

File: uploadNeuroDB/minc_deletion.pl [hunk 8]
```diff
@@ -1,4 +1,48 @@
...
+=head1 DESCRIPTION
+
+This program deletes MINC files from LORIS by:
+  - Moving the existing files (C<.mnc>, C<.nii>, C<.jpg>, C<.header>,
+    C<.raw_byte.gz>) to the archive directory: C</data/$PROJECT/data/archive/>
+  - Deleting all related data from C<parameter_file> & C<files> tables
+  - Deleting data from C<files_qcstatus> & C<feedback_mri_comments>
+    database tables if the C<-delqcdata> option is set. In most cases
+    you would want to delete this when the images change
+  - Deleting mri_acquisition_dates entry if it is the last file
```
Comment (cmadjar) [inline uploadNeuroDB/minc_deletion.pl]: code format for table mri_acquisition_dates

Comment (cmadjar) [reply uploadNeuroDB/minc_deletion.pl]: the last tiny change before I merge it :)

Comment (MounaSafiHarab) [reply uploadNeuroDB/minc_deletion.pl]: done

**Pull request merged**